src/util/component-types.js

```diff
@@ -73,7 +73,7 @@ export function findTypes(props, types=[]) {
 
 	// get elements from props (using type.propName)
 	const elementsFromProps = _.reduce(types, (acc, type) => {
-		if (!_.isEmpty(type.propName)) {
+		if (!_.isUndefined(type.propName)) {
```

Comment:
thanks for this...this is subtle but important to be able to pass null as a value